parser/parser.y

```diff
@@ -5486,12 +5498,21 @@ Statement:
 		// TODO: This is used to fix issue #320. There may be a better solution.
 		$$ = $1.(*ast.SubqueryExpr).Query.(ast.StmtNode)
 	}
+|	TraceStmt
 |	TruncateTableStmt
 |	UpdateStmt
 |	UseStmt
 |	UnlockTablesStmt
 |	LockTablesStmt
 
+TraceableStmt:
+	SelectStmt
+|	DeleteFromStmt
```

Comment:
I feel like only enable `SelectStmt` first and then support others? Any idea?

Reply:
This question seems not related to this PR.

Reply:
`SelectStmt` need to be taken care of immediately. Other kinds of queries is not that important.